openacademy/security/openacademy_security.xml

```diff
@@ -21,7 +21,6 @@
         <field name="perm_write" eval="False"/>
         <field name="perm_create" eval="True"/>
         <field name="perm_unlink" eval="False"/>
-        <field name="domain_force">[('responsible_id','=', False)]</field>
```

Comment:
A record rule without domain makes no sense

Reply:
Hi @ajqn9094,

Could you help @ejdecena with this point, please?

Reply:
I have already updated it.